### ISSUE TITLE
whitelist

### ISSUE THREAD

**Issue by birka1** [stamparm/maltrail#102]
hi 

is there a way to whitelist specific address to specific trail 

for example : 

whitelist ip 192.168.1.1 only for trail - direct exe file 

thanks
izik

**Comment by stamparm** [comment]
No. You can whitelist the IP address (or domain name) in several ways, but you can't whitelist it for specific "trails".

**Comment by birka1** [comment]
Is this feature will be in the future ? 
i think it's really added value feature

thanks

**Comment by stamparm** [comment]
I'll leave this issue opened for potential future implementation

**Comment by stamparm** [comment]
That's what the ignore rules are for: `data/ignore_events.txt` takes `src_ip;src_port;dst_ip;dst_port` lines, and `IGNORE_EVENTS_REGEX` matches the whole event, so you can silence one trail for one host without whitelisting the host entirely.
